tests/run_bar_tests.sh

```diff
@@ -0,0 +1,200 @@
+#!/bin/sh
+. tests/common.sh
+
+enter_suite bar
+
+TEST_ROOT="$ROOT/tests/bar"
+TEST_PATH="$TEST_ROOT/path"
+TEST_STATIC_ROOT="$ROOT/tests/test_bar"
+
+test -d "$TEST_ROOT" && rm -r "$TEST_ROOT"
+mkdir "$TEST_ROOT"
+cp -r "$TEST_STATIC_ROOT/path" "$TEST_ROOT"
+cp -r "$TEST_STATIC_ROOT/powerline" "$TEST_ROOT"
+
+export PYTHONPATH="$ROOT${PYTHONPATH:+:}$PYTHONPATH"
+
+ln -s "$(which "${PYTHON}")" "$TEST_PATH"/python
+ln -s "$(which sed)" "$TEST_PATH"
+ln -s "$(which cat)" "$TEST_PATH"
+ln -s "$(which mkdir)" "$TEST_PATH"
+ln -s "$(which basename)" "$TEST_PATH"
+ln -s "$TEST_PATH/lemonbar" "$TEST_PATH/bar-aint-recursive"
+
+DEPRECATED_SCRIPT="$ROOT/powerline/bindings/bar/powerline-bar.py"
+
+run() {
+	env -i \
+		LANG=C \
+		PATH="$TEST_PATH" \
+		XDG_CONFIG_HOME="$TEST_ROOT" \
+		XDG_CONFIG_DIRS="$TEST_ROOT/dummy" \
+		PYTHONPATH="$PYTHONPATH" \
+		TEST_ROOT="$TEST_ROOT" \
+		"$@" || true
+}
+
+display_log() {
+	local log_file="$1"
+	echo "$log_file:"
+	echo '============================================================'
+	cat -v "$log_file"
+	echo
+	echo '____________________________________________________________'
+}
+
+check_log() {
+	local log_file="$1"
+	local text="$2"
+	local warns="$3"
+	if test "$warns" = "warns" ; then
+		local warning="$(head -n1 "$log_file" | sed 's/.*://')"
+		local expwarning="The 'bar' bindings are deprecated, please switch to 'lemonbar'"
+		if test "x$warning" != "x$expwarning" ; then
+			echo "Got: $warning"
+			echo "Exp: $expwarning"
+			fail "warn" F "Expected warning"
+		fi
+		sed -r -i -e '1d' "$log_file"
+	fi
+	local line="$(head -n1 "$log_file")"
+	local linenum="$(cat "$log_file" | wc -l)"
+	if test $linenum -lt 5 ; then
+		fail "log:lt" F "Script was run not enough times"
+		return 1
+	elif test $linenum -gt 15 ; then
+		fail "log:gt" E "Script was run too many times"
+		return 1
+	fi
+  local expline="%{l}%{F#ffd0d0d0}%{B#ff303030} $text-left %{F-B--u}%{F#ff303030} %{F-B--u}%{r}%{F#ff303030} %{F-B--u}%{F#ffd0d0d0}%{B#ff303030} $text-right %{F-B--u}"
```

Comment:
I mean, in indentation. Though with most recent suggestion this line should not be different from line in af25681a8003e364e98f11717e62426abc184480.